main/src/ca/uwaterloo/flix/language/phase/Safety.scala

```diff
@@ -270,14 +284,46 @@ object Safety extends Phase[Root, Root] {
       case Polarity.Negative =>
         // Compute the free variables in the terms which are *not* bound by the lexical scope.
         val freeVars = terms.flatMap(freeVarsOf).toSet intersect quantVars
+        val errors = checkNegativeWildcards(terms, loc)
```

Comment:
@magnus-madsen This is the only new part of safety, I got a bit sidetracked looking at Intellij warnings. I can remove the reformatting if you prefer but maybe we could discuss some style guidelines in the compiler. Here I tried to eliminate unused variables mostly.

Reply:
Thats good! We are slowing moving towards removing unused vars-- like in Flix.

Reply:
Could be nice with Flix like _something variables so you can glance what information is being ignored.

Reply:
Yes. However the problem is that such names get outdated and are not updated during refactorings.